xblock/django/request.py

```diff
@@ -66,6 +65,9 @@ def __iter__(self):
             if key in self.UNPREFIXED_HEADERS or key.startswith('HTTP_'):
                 yield self._un_meta_name(key)
 
+    def __next__(self):
+        return self._meta.__next__()
+
```

Comment:
Does this change the behavior of this class?

If not, and this code is unused, could it be better to just leave the `pylint-disable` in place?